pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -978,6 +986,75 @@ func isGenericCSIPersistentVolumeClaim(pvc *v1.PersistentVolumeClaim, volInfos [
 	return false, nil
 }
 
+func (a *ApplicationRestoreController) getPVCFromObjects(objects []runtime.Unstructured, volumeBackupInfo *storkapi.ApplicationBackupVolumeInfo) (*v1.PersistentVolumeClaim, error) {
```

Comment:
I could similar definition in the kdmp driver as well, Can we move it to common util and use it.

Reply:
addressed.